worker/worker-execution/score-worker-execution-impl/src/test/java/io/cloudslang/worker/execution/services/ExecutionServiceTest.java

```diff
@@ -214,22 +255,22 @@ public void executeStepTest() throws InterruptedException {
 		executionService.executeStep(exe, executionStep);
 
 		Assert.assertEquals(0, exe.getPosition().longValue()); //position is still 0
-		Assert.assertTrue(exe.getSystemContext().hasStepErrorKey()); //there is error in context
+		assertTrue(exe.getSystemContext().hasStepErrorKey()); //there is error in context
 	}
 
 	@Test
 	public void executeNavigationTest() throws InterruptedException {
 		//Test no exception is thrown - all is caught inside
 		ExecutionStep executionStep = new ExecutionStep(EXECUTION_STEP_1_ID);
-		executionStep.setNavigation(new ControlActionMetadata("class", "method"));
+		executionStep.setNavigation(RUNTIME_EXCEPTION_METADATA);
 		executionStep.setNavigationData(new HashMap<String, Serializable>());
 
 		Execution exe = new Execution(0L, 0L, new HashMap<String,String>());
 
 		executionService.navigate(exe, executionStep);
 
-		Assert.assertEquals(null, exe.getPosition()); //position was changed to NULL due to exception
-		Assert.assertTrue(exe.getSystemContext().hasStepErrorKey()); //there is error in context
+        assertNull(exe.getPosition()); //position was changed to NULL due to exception
```

Comment:
is here an indentation problem?

Reply:
appears fine in the source file - actually did a double check after I committed. will edit here just in case